packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -3645,16 +3645,17 @@ The password specified in the value property must meet the default password poli
 
 ##### Hashed Password Object
 
-Specifies a hashed password that can be imported into Okta.  This allows an existing password to be imported into Okta directly from some other store.
-A hashed password may be specified in a Password Object when creating or updating a user, but not for other operations.  When updating a user with a hashed password the user must have the `STAGED` status.
+Specifies a hashed password to import into Okta. This allows an existing password to be imported into Okta directly from some other store. Okta supports the BCRYPT, SHA-512, SHA-256, SHA-1, and MD5 hashing functions for password import. A hashed password may be specified in a Password Object when creating or updating a user, but not for other operations.  See [Create User with Imported Hashed Password](#create-user-with-imported-hashed-password) for information on using this object when creating a user. When updating a user with a hashed password the user must have the `STAGED` status.
```

Comment:
The last sentence I struggled with a bit. There's nothing glaringly wrong with it, I just stumble over it each time I read it. I am suggesting the following, but take it or leave it if you like yours better:

When updating a user with a hash password, their status must be set to `STAGED`.

Reply:
Don't want to imply that should  do an action of setting status to staged, so have edited to: "When updating a user with a hashed password the user must be in the `STAGED` status."